rootex/core/renderer/constant_buffer.h

```diff
@@ -69,13 +70,13 @@ struct LightsInfo
 {
 	Vector3 cameraPos;
 	int pointLightCount = 0;
-	PointLightInfo pointLightInfos[4];
+	PointLightInfo pointLightInfos[MAX_POINT_LIGHTS];
```

Comment:
We will later make this a system setting after #247 gets merged, so we can control the number of lights from the editor

Reply:
issue is that this info is also used in shaders